alyssa_mckee/Django/belt_reviwer/apps/review_app/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from ..book_app.models import Book
+from ..login_registration.models import User
+# Create your models here.
+class ReviewManager(models.Manager):
+	def validate_review(self, data):
```

Comment:
Should add in another validation to make sure that the rating is a valid integer. Protect ourselves from malicious users.

Reply:
And also is a good preventative from having our app throw errors that would be unsightly.